### PR TITLE
Bphh 1848 story copying content to live mode review manager

### PR DESCRIPTION
**Pull request by bkmaibach**
## Description

- Add endpoint and api call for promote customization to live non-sandbox.
- Add confirmation modals for save and promote buttons
- update use-jurisdiction-template-version-customization.ts to handle sandbox changes
- add temporary sandbox ID selector and sandbox requirement template scoping

## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📦 Chore (Release)
- [ ] ✅ Test

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-1848

### CODE REVIEW

File: app/controllers/api/permit_applications_controller.rb [hunk 1]
```diff
@@ -340,12 +338,6 @@ def create_or_update_permit_block_status
 
   private
 
-  def determine_sandbox_id
-    return permit_application_params[:sandbox_id] if current_user.super_admin?
-
-    current_sandbox&.id
-  end
-
```
Comment (LaurelOlson) [inline app/controllers/api/permit_applications_controller.rb:348]: should also update the front end to no longer pass in `sandbox_id` as a param

File: app/frontend/components/domains/requirement-template/screens/jurisdiction-edit-digital-permit-screen/index.tsx [hunk 2]
```diff
@@ -37,6 +38,7 @@ export const formScrollToId = (id: string) => `${scrollToIdPrefix}${id}`
 export interface IJurisdictionTemplateVersionCustomizationForm {
   jurisdictionId?: string
   customizations: ITemplateCustomization
+  sandboxId?: string
```
Comment (LaurelOlson) [inline app/frontend/components/domains/requirement-template/screens/jurisdiction-edit-digital-permit-screen/index.tsx:41]: doesn't look like this is used anywhere?

File: config/locales/en.yml [hunk 3]
```diff
@@ -500,6 +501,12 @@ en:
       update_error:
         title: Error
         message: "Something went wrong updating Jurisdiction Customization"
+      promote_success:
+        title: ""
+        message: "Jurisdiction Template Customization promoted!"
+      promote_error:
+        title: Error
+        message: "Something went wrong promoting Jurisdiction Customization"
```
Comment (LaurelOlson) [inline config/locales/en.yml:509]: Double check casing here - these should probably use lowercase?

File: app/controllers/api/permit_applications_controller.rb (hunk 1 above)
Comment (bkmaibach) [reply app/controllers/api/permit_applications_controller.rb:348]: fixed in a later branch

**Review by LaurelOlson — APPROVED** [review]

**Pull request merged**